README.md

```diff
@@ -53,3 +53,31 @@ Use the following URL patterns to load data into databases:
 The docker images will take up more disk spaces as time goes on, you can delete those outdated docker images by using the following command:
 
 `docker rmi $(docker images -q -f "dangling=true")`
+
+This will remove everythin! (images, containers, volumes)
+`docker system prune -a`
+
+
+## Testing tips!
+
+1. Create a super user to test login. Run this command below. The password will be printed unless you specify it with --password. You can run this multiple times to change a password but you need to delete/modify super users via the /admin.
+
+`docker exec -it student_dashboard python manage.py createuser --superuser --username=root --email=root@example.edu`
+
+You can create regular users to test with without the superuser flag
+
+`docker exec -it student_dashboard python manage.py createuser --username=student --email=student@example.edu`
+
+Then this super user can login on the /admin URL
+
+2. Connect to the docker and edit some files!
+
+`docker exec -it student_dashboard /bin/bash`
+then install a text editor like vim
+`apt-get -y install vim`
+
+Then you can edit your files! (Probably in /dashboard/dashboard)
+
+3. Restart the gunicorn to read the configuration. This is useful to avoid a redeploy.
```

Comment:
Just configuration file changes, right? Will it pick up .py file changes?

Reply:
Yes this will reload .py files! This is how you can test things or hotfix (on the Openshift terminal for instance).

Configuration file changes (in the start.sh) cannot be tested this way. If you kill the process without -HUP it kills Openshift. It's possible to start up a temporary container to test changes to start.sh but I didn't document this here.